lauraguerra1/whats-cookin
# Feature/change icons

Description
- Change icons to match functionality
- Change the variable naming to match functionality
Test
- Ensure that the graph is functional after clicks.
- Check if add/remove works on both views with different screen sizes
